avail/src/main/kotlin/avail/anvil/projects/GlobalAvailConfigurationV1.kt

```diff
@@ -136,6 +148,23 @@ class GlobalAvailConfigurationV1
 					config.globalTemplates[name] = expansion.string
 				}
 			}
+			if (obj.containsKey(
```

Comment:
I got excited about this but actually JSON Object doesn't have this. It would be nice. We'd need to add the following function to `JSONObject`:
`operator fun contains (k: String): Boolean = k in map`
But you did teach me that this is a thing for maps! :)